lua/nodes/snippet.lua

```diff
@@ -335,7 +335,7 @@ function Snippet:indent(line)
 	self.indentstr = prefix
 	-- Check once here instead of inside loop.
 	if vim.o.expandtab then
-	    local tab_string = string.rep(" ", vim.o.tabstop)
+	    local tab_string = string.rep(" ", vim.o.shiftwidth or vim.o.tabstop)
```

Comment:
This only checks whether `shiftwidth` is set, which it always is (I think).
Maybe replace with `vim.o.shiftwidth == 0`

Reply:
Also check line 240 of this file, there is another check.